src/util.js

```diff
@@ -435,6 +435,13 @@ Oskari.util = (function () {
         return this.getColorBrightness(color) === 'light';
     };
 
+    util.hexToRgb = (hex) => {
```

Comment:
We should add some tests for this as util.js already has a test file.

Reply:
Didn't notice that util.js already had a hexToRgb function, so this is not needed.